shopping-list-web/src/utilities/httpMethods/recipeMethods.ts

```diff
@@ -0,0 +1,32 @@
+import IListItem from "../../ListItem/IListItem";
+import IRecipe from "../../Recipes/IRecipe";
+
+const objectToMap = (obj: Map<string, IRecipe>) =>
+  new Map<string, IRecipe>(Object.entries(obj));
+
+export const getRecipeData = async () => {
+  const response = await fetch("http://127.0.0.1:5000/recipes");
```

Comment:
Non blocker for the PR: Should the URL not be an environment variable or something?

Reply:
Yeah good point: https://github.com/srichardson-scottlogic/shopping-list/issues/27